src/webview/webViewEventHandlers.js

```diff
@@ -125,7 +130,8 @@ export type MessageListEvent =
   | MessageListEventReactionDetails
   | MessageListEventDebug
   | MessageListEventError
-  | MessageListEventMention;
+  | MessageListEventMention
+  | MessageListTime;
```

Comment:
Let's have this start with `MessageListEvent`, like the others. Then maybe have something a bit more descriptive; `MessageListEventTimeDetails` (looking at `MessageListEventReactionDetails`, above) or `MessageListEventPressTime`, maybe.

The names for the events going into and out of the WebView are very confusing; one part of #3913 is to fix that (see https://github.com/zulip/zulip-mobile/pull/3913/commits/6c9ff80864780c2e51164a642202d9de408d5ca5). But that PR has gone rather stale, and I'm hoping to get back to it before too long—likely after your PR lands, though.